master/reference/calicoctl/commands/node/run.md

```diff
@@ -78,7 +78,7 @@ Options:
                            [default: /var/log/calico]
      --node-image=<DOCKER_IMAGE_NAME>
                            Docker image to use for Calico's per-node container.
-                           [default: quay.io/calico/node:%s]
+                           [default: quay.io/calico/node:latest]
```

Comment:
This should still be `{{site.data.versions[page.version].first.components["calico/node"].version}}`